bifrost-fastify/index.ts

```diff
@@ -94,6 +94,12 @@ export const viteProxyPlugin: FastifyPluginAsync<
           { _pageId: string },
           typeof pageContextInit
         >(pageContextInit);
+        
+        if (req.url.endsWith('/')) {
```

Comment:
can we do this even before pageContextInit is generated?

Reply:
yeah good point, addressed.